tdrs-backend/tdpservice/settings/common.py

```diff
@@ -250,14 +250,20 @@ class Common(Configuration):
     # Sessions
     SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
     SESSION_COOKIE_HTTPONLY = True
+    SESSION_COOKIE_SECURE = True
+    SESSION_COOKIE_SAMESITE = 'None'
     SESSION_TIMEOUT = 30
 
     # The CSRF token Cookie holds no security benefits when confined to HttpOnly.
     # Setting this to false to allow the frontend to include it in the header
     # of API POST calls to prevent false negative authorization errors.
     # https://docs.djangoproject.com/en/2.2/ref/settings/#csrf-cookie-httponly
     CSRF_COOKIE_HTTPONLY = False
-    CSRF_TRUSTED_ORIGINS = ['.app.cloud.gov', '.acf.hhs.gov']
+    CSRF_TRUSTED_ORIGINS = ['.app.cloud.gov', 'tanfdata.acf.hhs.gov','api-tanfdata.acf.hhs.gov']
+    CRSF_COOKIE_SECURE = True
+    CRSF_COOKIE_SAMESITE = 'None'
+    CRSF_HEADER_NAME = 'HTTP_X_CSRFTOKEN'
+
```

Comment:
Same thing here with the cookies, we do this stuff our self in the middleware extension we do.